File: C#HandsOn/Ex11/Sales.cs
using System;

namespace Ex11
    {
        public class Sales :Employee
        {
            //変数宣言
            int customerCount = 0;

            public Sales()
            {
                
            }
            public Sales(string lname,string fname,int n,int count) 
                : base(lname,fname,n)
            {
                customerCount = count;
            }

            public override void PrintInfo()
            {
                base.PrintInfo();
                Console.WriteLine("顧客数："+ customerCount);
            }
    }
    }

File: VBHandsOn/Ex11/Sales.vb
Public Class Sales
    Inherits Employee
    'フィールド
    Dim customerCount As Integer

    'コンストラクタ
    Public Sub New()

    End Sub
    Public Sub New(lname As String, fname As String, n As Integer, count As Integer)
        MyBase.New(lname, fname, n)
        customerCount = count
    End Sub

    'メソッド
    Public Overrides Sub PrintInfo()
        MyBase.PrintInfo()
        Console.WriteLine("顧客数：" & customerCount)
    End Sub

End Class
